docs/docs/tutorial/creating-a-source-plugin/part-5/index.mdx

```diff
@@ -0,0 +1,329 @@
+---
+title: "Part 5: Incremental Builds"
+---
+
+import { LinkButton } from "gatsby-interface"
+import Collapsible from "@components/collapsible"
+import { MdArrowForward } from "react-icons/md"
+
+## Introduction
+
+The term "Incremental Builds" is not specific to Gatsby as it's more a concept than a specific technology or implementation. Generally speaking it means that on subsequent runs of an action data is only changed incrementally, in contrast to having to do a full action each time. This is often achieved by populating a cache on the first run, listening for changes, and then only updating what has changed.
+
+Gatsby supports Incremental Builds to varying degrees and in different places:
+
+- The open source framework incrementally builds various steps in its build pipeline without requiring any special setup from users. These optimizations are a more generic approach and work with every site, regardless of their use case or sources.
+- Often times a large portion of the build is spent on sourcing data from remote APIs through Gatsby source plugins. Source plugin authors can write their plugin in such a way that on subsequent runs data sourcing is only happening incrementally. This behavior is called [Delta updates](#delta-updates) in this part of the tutorial. This optimization can further be supercharged if the source plugin is approved for [Cloud Builds](https://support.gatsbyjs.com/hc/en-us/articles/360053099253-Gatsby-Builds-Full-Incremental-and-Cloud) on Gatsby Cloud.
+- Source plugins can also create/update/delete GraphQL nodes in an incremental fashion instead of recreating them all on every run (`createNode`, `touchNode`, and `deleteNode`). This functionality is directly coupled with Delta updates.
+
+By the end of this part of the tutorial, you will be able to:
+
+- Use Node APIs to optimize node creation
+- Understand in more detail how delta updates work
```

Comment:
Maybe something like this would work well here? Especially because review questions are included at the end of each part.
```suggestion
- Describe in more detail how delta updates work
```